website/server/controllers/api-v3/quests.js

```diff
@@ -76,6 +76,7 @@ api.inviteToQuest = {
     if (!user.items.quests[questKey]) throw new NotAuthorized(res.t('questNotOwned'));
     if (user.stats.lvl < quest.lvl) throw new NotAuthorized(res.t('questLevelTooHigh', { level: quest.lvl }));
     if (group.quest.key) throw new NotAuthorized(res.t('questAlreadyUnderway'));
+    if (quest.previous && !user.achievements.quests[quest.previous]) throw new NotAuthorized(res.t('mustComplete', {quest: quest.previous}));
```

Comment:
Is this necessary? I mean the check is already done when the quest is bought so this should not be necessary

Reply:
I thought that too but without it we'd be left with a (potentially very small) hole where people who have bought quests that they shouldn't be able to would be able to start them before they were supposed to. I'm fine with removing it, but the issue said to change it as well, so I put it in

Reply:
Opinions @Alys ? i feel like it would be better to leave the loophole and impose the restrictions only for newly bought quests

Reply:
@paglias If you'd prefer it that way, that's fine with me. @fentie is right that's it's a small loophole - not very important.

Reply:
Ok, removed and pushed.